tiffslide/tiffslide.py

```diff
@@ -89,6 +90,7 @@ def __init__(
         *,
         tifffile_options: dict[str, Any] | None = None,
         storage_options: dict[str, Any] | None = None,
+        series_idx: int | str = 'auto',
```

Comment:
I would suggest leaving the series_idx selection out of this PR for now, and just focus on providing openslide compatible scn handling.

Could you remove the series_idx kwarg etc from here, and basically default to the 'auto' selection in the code below?